.github/workflows/main.yml

```diff
@@ -108,7 +108,9 @@ jobs:
             cd forte-wrappers
             for d in ${{ matrix.notebook-details.dep }}; do pip install "src/"$d; done
             cd ..
-            pip install --progress-bar off .[data_aug,ir,remote,audio_ext,stave,models,test,wikipedia,nlp,extractor]
+            sudo apt-get install automake ca-certificates g++ git libtool libleptonica-dev make pkg-config
```

Comment:
Are they all required to install `tesseract-ocr`?

Reply:
yeah, probably should create matrix runs to separate the dependencies.